addons/web/static/tests/fields/relational_fields_tests.js

```diff
@@ -4314,6 +4314,97 @@ QUnit.module('relational_fields', {
         form.destroy();
     });
 
+    QUnit.test('onchange many2many in one2many list editable', function (assert) {
```

Comment:
i don't know what exactly your test is supposed to test, but it passes in the current master, so it doesn't check at all your new development, and it should